### PR TITLE
CO-81 Query to retrieve all streets.

### PR DESCRIPTION
**Pull request by JeppeTrip**

### CODE REVIEW

File: backend/Application.UnitTests/Streets/Queries/GetStreetsQueryTest.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+namespace Application.UnitTests.Streets.Queries.GetStreets
+{
+  [Collection("QueryTests")]
+  public class GetStreetsQueryTest : CommandTestBase
+  {
+    private readonly ApplicationDbContext _context;
+    private readonly IMapper _mapper;
+
+    public GetStreetsQueryTest(QueryTestFixture fixture)
```
Comment (CSCITM) [inline backend/Application.UnitTests/Streets/Queries/GetStreetsQueryTest.cs:18]: You should test correct paging too.

File: backend/Application/Streets/Queries/GetStreetsQuery.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+namespace Application.Streets.Queries.GetStreets
+{
+  public class GetStreetsQuery : IPageRequest<StreetDto>, IPageBody<Street, string>
+  {
+    public int Size { get; set; }
+    public int? Skip { get; set; }
+    public string Needle { get; set; }
+
+    public string GetNewNeedle(IQueryable<Street> query)
```
Comment (CSCITM) [inline backend/Application/Streets/Queries/GetStreetsQuery.cs:21]: Any reason these are methods when they are only used once?

File: backend/Web/Controllers/StreetController.cs [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  public class StreetController : ApiControllerBase
+  {
+    [HttpGet]
+    public async Task<ActionResult<PageResult<StreetDto>>> Get(
+     [FromQuery] string needle, [FromQuery] int size, [FromQuery] string sortBy, [FromQuery] int? skip = 0
+   )
+    {
+      try
+      {
+        switch (sortBy)
```
Comment (CSCITM) [inline backend/Web/Controllers/StreetController.cs:23]: This should be handled in the command.

File: backend/Application/Streets/Queries/GetStreetsQuery.cs (hunk 2 above)
Comment (JeppeTrip) [reply backend/Application/Streets/Queries/GetStreetsQuery.cs:21]: They are from the IPageBody interface definition, and for logical subdivision of functionality. 
@objmj-itminds might have a more satisfying answer than this.

File: backend/Web/Controllers/StreetController.cs (hunk 3 above)
Comment (JeppeTrip) [reply backend/Web/Controllers/StreetController.cs:23]: I will be happy to move it or remove it completely, but I did follow the example from `backend/Web/Controllers/ExampleEntityController.cs`.
@objmj-itminds do you agree that I should handle this elsewhere?

Comment (objmj-itminds) [reply backend/Web/Controllers/StreetController.cs:23]: When there is only one case no need for a switch.

**Review by objmj-itminds — APPROVED** [review]

**Pull request merged**